components/service/pocket/src/main/java/mozilla/components/service/pocket/data/PocketGlobalVideoRecommendation.kt

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket.data
+
+/**
+ * A recommended video as returned from the Pocket Global Video Recommendation endpoint v2.
+ */
+data class PocketGlobalVideoRecommendation(
```

Comment:
This looks like a class were we would want to document every `@property`? Does the endpoint have an API documentation where we could copy it from? At least a field like `publishedTimestamp: String` in itself doesn't say much.

Reply:
> This looks like a class were we would want to document every `@property`?

Good point: I forgot that I was creating a public API. :)

> Does the endpoint have an API documentation where we could copy it from? At least a field like `publishedTimestamp: String` in itself doesn't say much.

Unfortunately, no – these are internal APIs with minimal documentation. There are no details on what each field represents, beyond the occassional incidental comment, and just JSON dumps of examples of the API output.

Reply:
I don't actually know what `publishTimestamp` is, tbh. I didn't think it was worth asking Pocket about unless someone needed that info so I put in:
```kotlin
/**
 * @property publishedTimestamp unknown: please ask for clarification if needed. May be "0".
```

Reply:
I added kdoc to all of the other properties (and added kdoc to the `Authors` field too).